packages/docs/versioned_docs/version-2.0.0-alpha-3/testnet-deployment.md

```diff
@@ -35,3 +35,7 @@ This URL can be directly used in a [network prover](circuits-api/network-prover)
 :::danger
 These keys have not had a secure phase 2 trusted setup. **Use these keys at your own risk.**
 :::
```

Comment:
Does this mean that the deployed contract is using these keys despite this noted danger? Is this ok because we're on testnet, and the warning is so that devs don't use them in another context, for some other contract?

Reply:
Yes. When we do a trusted setup each participant creates toxic waste. This toxic waste is a sequence of numbers that can be used to forge proofs. The danger is that we (the unirep team) are the only ones to participate in the ceremony, so there isn't a very strong guarantee that we destroyed the toxic waste.

So the threat is that we could theoretically forge proofs in systems that use the keys we provide. To us this is a small threat because we know how they keys were generated, but to anyone else has no reason to trust us.